[ADD] base_bank_branch: init (#8)
focusate/misc: base_bank_branch/README.rst

> Bank Branches

Reviewer: Hmm.. Isn't 'hierarchy' more suitable here?
```suggestion
Bank Hierarchy
```

Author: Hierarchy is too generic. It says nothing about branches, where branch is real term regarding banks structure.

#############

> Adds branches support for banks. When creating :code:`Bank` can now check :code:`Is Branch` option, which then requires to specify its bank as parent.

Reviewer: ```suggestion
Adds branches support for banks. When creating :code:`Bank` can now check :code:`Is Branch` option, which then requires to specify it's :code:`Main Bank`.
```
